uint16[] ids
----
ServoInfo[] servos